### PR TITLE
Default firewallrules

### PR DESCRIPTION
**Pull request by NaveenSidhartha**
updated default firewall rules as per docs and added new fields tests

### CODE REVIEW

File: data_blob.py [hunk 1]
```diff
@@ -443,13 +443,17 @@ def data_blob():
...
+    inst_conf_pras(48)
+    inst_conf_pv6s(49)
+    inst_conf_cmon(50)
+    ping_ipv4___pe(51)
+    ping_ipv4__cpe(52)
+    ping_ipv4_8888(53)
+    ping_ipv6___pe(54)
+    ping_ipv6__cpe(55)
+    ping_ipv6_8888(56)
+    ping_dns__ggle(57)
```
Comment (maria-walsh) [inline data_blob.py:456]: The `number_of_tests` in the `sql_utils` for building the `test_results` map needs to be updated when adding/removing tests.

File: data_blob.py [hunk 2]
```diff
@@ -443,13 +443,17 @@ def data_blob():
...
     inst_conf_6pmm(46)
-    ping_ipv4___pe(47)
-    ping_ipv4__cpe(48)
-    ping_ipv4_8888(49)
-    ping_ipv6___pe(50)
-    ping_ipv6__cpe(51)
-    ping_ipv6_8888(52)
-    ping_dns__ggle(53)
+    inst_conf_dnss(47)
+    inst_conf_pras(48)
```
Comment (maria-walsh) [inline data_blob.py:447]: This test is false in all the cop podnet a and b fail maps. Should it be true?

**Review by maria-walsh — CHANGES_REQUESTED** [review]

File: data_blob.py (hunk 2 above)
Comment (NaveenSidhartha) [reply data_blob.py:447]: oh yes its not required for cop only pods, reseting it.

File: data_blob.py (hunk 1 above)
Comment (NaveenSidhartha) [reply data_blob.py:456]: my bad adding it

**Review by maria-walsh — APPROVED** [review]

**Pull request merged**